cli/README.md

```diff
@@ -62,17 +81,21 @@ graph TD
     G --> |no| I(Use the default branch)
 ```
 
-As you can see, the current git branch is used in the first two cases in the algorithm:
+The current git branch is used in the first two cases in the algorithm:
 
-- If a git branch has been explicitely mapped to a xata branch with `xata branches link` for example, the associated branch is used.
-- Else, if a xata branch with the same name as the git branch exists, it is used.
+- if a git branch has been explicitly mapped to a Xata branch with `xata branches link` for example, the associated branch is used
+- otherwise, if a Xata branch with the same name as the git branch exists, we use that.
 
-If none of them apply, a fallback branch is used. This is useful to avoid connecting to your production database. If you are developing a feature that doesn't require schema changes, you'll have a git branch, but not an associated xata branch. In this case to prevent using the default xata branch, that could be production, you can specify in your `.env` file a `XATA_FALLBACK_BRANCH`. You can have a personal one with your username for example.
+If none of them apply, the **fallback branch** is used. This is a safety measure to avoid connecting to your production database. If you are developing a feature that doesn't require schema changes, you'll have a git branch, but not an associated Xata branch. In this case to prevent using the default Xata branch–which could be production– we allow specifying a `XATA_FALLBACK_BRANCH` in your local `.env` file. You can have a personal branch with your username for example.
```

Comment:
Forgive me—what exactly is the suggestion here? Is it about spacing around the em-dash?

As far as I know in English, there are no hard rules about this. Sometimes publications either pad em-dashes with spaces, but others don’t depending on literary preferences. 

So far, we’ve opted to omit them for the same reasons Merriam-Webster did (ref: https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use) but since there’s no hard rules, I don’t have a strong opinion. Happy to have a discussion and a proposal around this if you like, but changing it here in this case will make the rest of the docs and other written media inconsistent and probably is a bigger task.

Reply:
Cool. I didn't know it can be used without spaces. I've always seen it with spaces, I think. To me it's easier to read, but not strong opinions.

Nitpick: the character to use seems to be — not - acording to that article.

Reply:
Fuck, you're right. These are en-dashes, not em-dashes. Great catch! Thanks! Fixed in 75e5b6c6ecd2459e69368c3b7461d6ee64bc155d